packages/sommelier.rb

```diff
@@ -345,9 +420,10 @@ def self.build
         RESTARTSOMMELIEREOF
 
         # start sommelier from bash.d, which loads after all of env.d via #{CREW_PREFIX}/etc/profile
-        @bashd_sommelier = <<~BASHDEOF
+        File.write 'bash.d_sommelier', <<~BASHDSOMMELIEREOF
           source #{CREW_PREFIX}/bin/startsommelier
-        BASHDEOF
+          pgrep Xwayland &>/dev/null && source #{CREW_PREFIX}/etc/sommelierrc
```

Comment:
Replace `&> /dev/null` with `2>> #{CREW_PREFIX}/var/log/sommelier.log`?

Reply:
That will only put the PID of Xwayland into the log... is that useful?

Reply:
made the change...